docs(webviews API): revert to old version and update with recent changes. (#4400)
wix/Detox: docs/api/webviews.md

However, elements within web views are not native components, making direct interaction through Detox challenging.
To address this, Detox offers a suite of matchers, actions, and expectations designed for interacting with content inside web views.

## Locating Elements in Web Views

### `web.element(matcher)`

In cases where there's only one web view present on the screen, you may use the `web.element()` function, paired with [web view matchers], to reference elements within the web view.
Upon obtaining the element reference, you can utilize web view actions and expectations on the webView element.

```js
const innerElement = web.element(by.web.id('inner_element_identifier'));
await expect(innerElement).toHaveText('Hello World!');
```

> The operation above explains how to locate an inner element using its HTML id attribute and how to verify its text content.

Reviewer: ```suggestion
In the example above, we locate an inner element by its `id` HTML attribute and verify its text content.
```

### `web(nativeMatcher).element(matcher)`

> For screens with several web views, it's first necessary to identify a specific web view using a native matcher.

Reviewer: If you have multiple web views on the screen, you must locate a specific web view first by using a [native matcher][native matchers], e.g.:

```js
const myWebView = web(by.id('webview_identifier'));
```

Following that, you can locate the element within the identified web view:

```js
const innerElement = myWebView.element(by.web.id('inner_element_identifier'));
await expect(innerElement).toHaveText('Hello World!');
```

Following that, you can locate the element within the identified web view.

If you have multiple web views on the screen, you must locate a specific web view first by using a [native matcher][native matchers], e.g.:

```js
const myWebView = web(by.id('webview_identifier'));
const innerElement = myWebView.element(by.web.id('inner_element_identifier'));
await expect(innerElement).toHaveText('Hello World!');
```

In the example above:

1. We use `web()` function and [`by.id()`] matcher to locate a web view by its accessibility identifier.
1. We use `myWebView.element()` method and [`by.web.id()`] web matcher to find an HTML element inside that web view.
1. We run the same expectation (to have text) as in the previous example.

> ### `web(nativeMatcher).atIndex(index).element(matcher)`

Reviewer: ```suggestion
### `web(nativeMatcher).atIndex(index).element(matcher)` (iOS only)
```

If you have multiple web views on the screen and you want to interact with a specific web view, you can use the `atIndex()` method to choose the web view at the specified index.

```js
const myWebView = web(by.id('webview_identifier').atIndex(1));
const innerElement = myWebView.element(by.web.id('inner_element_identifier'));
await expect(innerElement).toHaveText('Hello World!');
```

In the example above, we use `atIndex()` to select the second web view on the screen (that has the specified accessibility identifier) and then locate an HTML element inside that web view.

> :::note

Reviewer: I think it's best to move this note under the heading.
